src/components/ArtifactDetailPage/ArtifactDetailPage.tsx

```diff
@@ -0,0 +1,139 @@
+import { BuildName } from 'components/BuildName/BuildName';
+import { ContentBox } from 'components/ContentBox/ContentBox';
+import { DownloadLink } from 'components/DownloadLink/DownloadLink';
+import { ArtifactQualityLabelMapper } from 'components/LabelMapper/ArtifactQualityLabelMapper';
+import { ArtifactRepositoryTypeLabelMapper } from 'components/LabelMapper/ArtifactRepositoryTypeLabelMapper';
+import { Toolbar } from 'components/Toolbar/Toolbar';
+
+import { calculateFileSize, createDateTime } from 'utils/utils';
+
+export const ArtifactDetailPage = () => {
+  const { serviceContainerArtifact } = useServiceContainerArtifact();
+
+  return (
+    <Grid hasGutter>
+      <GridItem xl={12} xl2={6}>
+        <Toolbar>
+          <ToolbarItem>
+            <TextContent>
+              <Text component={TextVariants.h2}>Details</Text>
+            </TextContent>
+          </ToolbarItem>
+        </Toolbar>
+        <ContentBox borderTop padding contentBoxHeight={`calc(100% - 78px)`}>
+          <Attributes>
+            <AttributesItem title={artifactEntityAttributes.filename.title}>
+              {serviceContainerArtifact.data?.filename}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.purl.title}>{serviceContainerArtifact.data?.purl}</AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.size.title}>
+              {serviceContainerArtifact.data?.size && calculateFileSize(serviceContainerArtifact.data.size)}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.artifactQuality.title}>
+              <ArtifactQualityLabelMapper quality={serviceContainerArtifact.data?.artifactQuality} />
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.importDate.title}>
+              {serviceContainerArtifact.data?.importDate &&
+                createDateTime({ date: serviceContainerArtifact.data.importDate }).custom}
+            </AttributesItem>
+          </Attributes>
+        </ContentBox>
+      </GridItem>
+
+      <GridItem xl={12} xl2={6}>
+        <Toolbar>
+          <ToolbarItem>
+            <TextContent>
+              <Text component={TextVariants.h2}>Checksums</Text>
+            </TextContent>
+          </ToolbarItem>
+        </Toolbar>
+        <ContentBox borderTop padding contentBoxHeight={`calc(100% - 78px)`}>
+          <Attributes>
+            <AttributesItem title={artifactEntityAttributes.md5.title} forceStringWrap>
+              {serviceContainerArtifact.data?.md5}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.sha1.title} forceStringWrap>
+              {serviceContainerArtifact.data?.sha1}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.sha256.title} forceStringWrap>
+              {serviceContainerArtifact.data?.sha256}
+            </AttributesItem>
```

Comment:
Do you think it could be useful to apply `CopyToClipboard` component here so that users can easily copy it if needed?

Reply:
Yes I applied it, but do you find the inline or normal variant better?